enterprise/backend/src/metabase_enterprise/metabot_v3/client.clj

```diff
@@ -162,13 +176,13 @@
           canceled (atom nil)]
       (metabot-v3.context/log (:body response) :llm.log/llm->be)
       (log/debugf "Response from AI Proxy:\n%s" (u/pprint-to-str (select-keys response #{:body :status :headers})))
-      (when-not (= (:status response) 200)
+      (when (< 299 (:status response))
```

Comment:
Hmm feels weird to relax status to all 100s and 200s, and the code looks a bit "weird" too.

I guess this is to whitelist partial content? I think lets just have an explicit set of allowed response codes, because IMHO even a 304 would make more sense than say a 207 (multi status)